starter-code/functions-and-arrays.js

```diff
@@ -20,9 +16,23 @@ var words = [
   "orchard",
   "crackpot"
 ];
+
+var lgth = 0;
+
+function findLongestWord (words) {
+ for (i=0; i<words.length; i++){
+   if (words[i].length > lgth){
+     lgth = words[i].length;
```

Comment:
You don't need to store the length of the longest word because you can access it with `longest.length`